zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -100,6 +100,12 @@
                   {{note.info.cron}}
                 </p>
               </div>
+              <div>
+                <span>- Release resource after cron execution </span>
```

Comment:
Since 'resource' is not a term used in Zeppelin, i think 

"Restart interpreter (release resource) after cron execution"

is more clear.